blocks/top-table-list-block/features/top-table-list/_children/large.jsx

```diff
@@ -0,0 +1,210 @@
+	Overline,
+	Heading,
+	Paragraph,
+	Date as DateDisplay,
+	formatAuthors,
+	Attribution,
+	Separator,
+	usePhrases,
+	Video,
+	Divider,
+} from "@wpmedia/arc-themes-components";
+
+const BLOCK_CLASS_NAME = "b-top-table-list-large";
+
+const getType = (type, content) => (content?.type === type ? content : undefined);
+
+const Large = (props) => {
+	const {
+		element,
+		customFields: {
+			showBylineLG,
+			showDateLG,
+			showDescriptionLG,
+			showHeadlineLG,
+			showImageLG,
+			showOverlineLG,
+			playVideoInPlaceLG,
+			showBottomBorderLG,
+			imageRatioLG = "3:2",
+		},
+		fallbackImage,
+	} = props;
+	const { arcSite } = useFusionContext();
+
+	const showBottomBorder = typeof showBottomBorderLG === "undefined" ? true : showBottomBorderLG;
+
+	const {
+		dateLocalization: { language, timeZone, dateTimeFormat } = {
+			language: "en",
+			timeZone: "GMT",
+			dateTimeFormat: "LLLL d, yyyy 'at' K:m bbbb z",
+		},
+	} = getProperties(arcSite);
+
+	const displayDate = localizeDateTime(
+		new Date(element?.display_date),
+		dateTimeFormat,
+		language,
+		timeZone
+	);
+	const phrases = usePhrases();
+
+	const videoOrGalleryContent =
+		getType("video", element) ||
+		getType("gallery", element) ||
+		getType("video", element?.promo_items?.lead_art) ||
+		getType("gallery", element?.promo_items?.lead_art);
+
+	const labelIconName = {
+		gallery: "Camera",
+		video: "Play",
+	}[videoOrGalleryContent?.type];
+
+	const labelIconText = {
+		gallery: phrases.t("global.gallery-text"),
+		video: phrases.t("global.video-text"),
+	}[videoOrGalleryContent?.type];
+
+	const {
+		display: labelDisplay,
+		url: labelUrl,
+		text: labelText,
+	} = (element?.label && element?.label?.basic) || {};
+
+	const { _id: sectionUrl, name: sectionText } =
+		element?.websites?.[arcSite]?.website_section || {};
+
+	let [overlineText, overlineURL] = [sectionText, sectionUrl];
+	if (element?.owner?.sponsored) {
+		overlineText = element?.label?.basic?.text || phrases.t("global.sponsored-content");
+		overlineURL = null;
+	} else if (labelDisplay) {
+		[overlineText, overlineURL] = [labelText, labelUrl];
+	}
+
+	const contentAuthors =
+		showBylineLG && element?.credits?.by?.length > 0
+			? formatAuthors(element.credits.by, phrases.t("global.and-text"))
+			: null;
+	const contentDate = showDateLG ? element?.display_date : null;
+	const contentDescription = showDescriptionLG ? element?.description?.basic : null;
+	const contentHeading = showHeadlineLG ? element?.headlines?.basic : null;
+	const contentOverline = showOverlineLG ? overlineText : null;
+	const contentUrl = element?.websites?.[arcSite]?.website_url;
+	const embedMarkup = playVideoInPlaceLG && getVideoFromANS(element);
+	const promoImageURL = showImageLG && (getImageFromANS(element)?.url || fallbackImage);
+
+	const imageParams = element
+		? {
+				ansImage: getImageFromANS(element),
+				alt: element?.headlines?.basic || "",
+				aspectRatio: imageRatioLG,
+				resizedOptions: {
+					smart: true,
```

Comment:
Don't we need an `auth` token here?

Reply:
for all the images, that is...

Reply:
It's handled within the `<Image />` component now if you use the `ansImage` prop